sdk/core/azure-core/inc/http/httppolicy.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "context.hpp"
+#include "http/request.hpp"
+#include "http/response.hpp"
+
+namespace Azure { namespace Core { namespace Http {
+
+  class HttpPolicy
+  {
+  public:
```

Comment:
Since this class can be derived, I think you need to prohibit public copy constructor and assignment